ssh_client.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+import paramiko
+
+
+def ssh_command(hostname, username, password, paramiko, command):
```

Comment:
@sbulage please review other PR https://github.com/kedark3/check-rhv/pull/8 and rename/rebase the ssh_client.py file appropriately, we want to call it `utils.py` and put all utilities in there:)

Reply:
Renamed File to `utils.py`